SafetyFirst/Set.fs

```diff
@@ -0,0 +1,220 @@
+module SafetyFirst.Set
+
+module NonEmpty = 
+
+  /// <summary>
+  /// Creates a new NonEmptySet with the provided head and tail.  
+  /// The tail is constrained to be finite.  If the tail is infinite,
+  /// use Seq.NonEmpty.create instead.
+  /// </summary>
+  let create head tail : NonEmptySet<_> = NonEmpty (set (Seq.append [head] tail)) 

```

Comment:
is it more performant or clrearer to have:
`NonEmpty (Set.add head (set tail))`
?

Reply:
Not sure, but it isn't any less clear.  I'll switch it over.

Reply:
✔️